scripts/ble_library.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/python
+"""
+SPDX-License-Identifier: Apache-2.0
+Copyright (c) 2021 STMicroelectronics.
+This script updates ble library
+"""
+
+import sys
+import subprocess
+from pathlib import Path
+import logging
+from jinja2 import Environment, FileSystemLoader
+
+include_dir = ["stm32wb/hci"]
+
+file_list = [
+    "Middlewares/ST/STM32_WPAN/ble/core/ble_bufsize.h",
+    "Middlewares/ST/STM32_WPAN/interface/patterns/ble_thread/hw.h",
+    "Middlewares/ST/STM32_WPAN/interface/patterns/ble_thread/shci/shci.c",
+    "Middlewares/ST/STM32_WPAN/interface/patterns/ble_thread/shci/shci.h",
+    "Middlewares/ST/STM32_WPAN/interface/patterns/ble_thread/tl/hci_tl.h",
+    "Middlewares/ST/STM32_WPAN/interface/patterns/ble_thread/tl/mbox_def.h",
+    "Middlewares/ST/STM32_WPAN/interface/patterns/ble_thread/tl/shci_tl_if.c",
+    "Middlewares/ST/STM32_WPAN/interface/patterns/ble_thread/tl/shci_tl.c",
+    "Middlewares/ST/STM32_WPAN/interface/patterns/ble_thread/tl/shci_tl.h",
+    "Middlewares/ST/STM32_WPAN/interface/patterns/ble_thread/tl/tl.h",
+    "Middlewares/ST/STM32_WPAN/interface/patterns/ble_thread/tl/tl_mbox.c",
+    "Middlewares/ST/STM32_WPAN/stm32_wpan_common.h",
+    "Middlewares/ST/STM32_WPAN/utilities/dbg_trace.h",
+    "Middlewares/ST/STM32_WPAN/utilities/stm_list.c",
+    "Middlewares/ST/STM32_WPAN/utilities/stm_list.h",
+    "Middlewares/ST/STM32_WPAN/utilities/utilities_common.h",
+    "Projects/P-NUCLEO-WB55.Nucleo/Applications/BLE/BLE_Hid/Core/Inc/app_common.h",
+    "Projects/P-NUCLEO-WB55.Nucleo/Applications/BLE/BLE_Hid/Core/Inc/app_conf.h",
+    "Projects/P-NUCLEO-WB55.Nucleo/Applications/BLE/BLE_Hid/Core/Inc/hw_if.h",
+    "Projects/P-NUCLEO-WB55.Nucleo/Applications/BLE/BLE_Hid/STM32_WPAN/Target/"
+    + "hw_ipcc.c",
+    "Projects/P-NUCLEO-WB55.Nucleo/Applications/BLE/BLE_Hid/STM32_WPAN/App/"
+    + "tl_dbg_conf.h",
+]
```

Comment:
It is not really helpful is this case because most of the case I ake only part of a directory. 
It could be useful only for  "Middlewares/ST/STM32_WPAN/interface/patterns/ble_thread/shci/" .. but It contains only to file
So I keep as is, which is at least explicit on the copied files.